### PR TITLE
Refactoring controllers

### LINKED ISSUE #527: Move some controller code to models/Refactor models
Hey @jywarren I have been going through the controllers and could not find code that could potentially belong to the model. Refactoring some code though. Any pointers of code you had in mind for this and maybe I could start from there? Thanks

### PR DESCRIPTION
**Pull request by cesswairimu**
Fixes #527 

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

File: app/controllers/maps_controller.rb [hunk 1]
```diff
@@ -135,13 +120,13 @@ def destroy
...
   def images
-    warpables = []
-    @map.warpables.each do |warpable|
-      warpables << warpable
-      warpables.last[:nodes] = warpable.nodes_array
-      warpables.last.src = warpable.image.url
-      warpables.last.srcmedium = warpable.image.url(:medium)
-    end
+    warpables = @map.warpables
+    # @map.warpables.each do |warpable|
```
Comment (cesswairimu) [inline app/controllers/maps_controller.rb]: @jywarren I couldn't find where these values were being used but wanted to check before I remove crucial code. Let me know if its safe to remove this.Thanks

**Comment by jywarren** [comment]
I think they're used by the front-end JS
https://github.com/publiclab/Leaflet.DistortableImage/ in code in
/app/assets/javascripts/... to find the corners of each image. So I believe
they are necessary! Try opening a map to see if this action is hit? Thanks
Cess!

**Comment by cesswairimu** [comment]
@publiclab/reviewers please review

**Comment by jywarren** [comment]
Awesome as usual Cess!!!! Merging now!

**Pull request merged**